scripts/triggers-end-to-end-tests/functions/index.js

```diff
@@ -1,15 +1,20 @@
 const admin = require("firebase-admin");
 const functions = require("firebase-functions");
+// TODO(taeold): firebase-functions/lib path is unsupported, but this is the only way to access v2 namesapce in Node10
+const functionsV2 = require("firebase-functions/lib/v2");
```

Comment:
Yet this will throw in Node 12. I would at least make this a try require for both formats (prefer the correct one)